ppoprf/src/ppoprf.rs

```diff
@@ -271,11 +445,17 @@ impl Server {
     let mut proof = None;
     if verifiable {
       let public_value = self.public_key.get_combined_pk_value(md)?;
-      proof = Some(ProofDLEQ::new(
+      /*proof = Some(ProofDLEQ::new(
         &tagged_key,
         &public_value.into(),
         &eval_point,
         &point,
+      ));*/
```

Comment:
Please remove the commented code. The differences are pretty obvious from the function signature change, and there's git history to compare with.